### PR TITLE
Release 0.0.3

### PR DESCRIPTION
**Pull request by zippy**
- [x] split out contributor guidelines into separate file
- [x] test that readme dev usage & installation instructions work
- [x] update version numbers
- [x] include instructions for n3h installation and usage.
- [x] documentation and docs references working (@Connoropolous?)
- [x] nodejs references correct in scaffold (see #838)
- [x] add change log
- [ ] update travis auto-build for two binaries (hc and holochain_container) instead of just one.

### CODE REVIEW

**Comment by zippy** [comment]
I was just making room for all the things you want to make sure get in!

**Comment by Connoropolous** [comment]
Actually yeah @zippy the only things I had targeted in the API ref that aren't in are a couple of lines describing some of the enums: 
https://developer.holochain.org/api/latest/hdk/#enums

File: README.md [hunk 1]
```diff
@@ -63,29 +60,38 @@ If you need to implement your own container, [container_api](container_api) shou
...
 
 **The following instructions are for developing Holochain Core or the HDK itself**
 
-There are two approaches to building and testing Holochain, using `make` or using `docker`:
+There are two components needed currently to run Holochain applications, the core (what's in this repo) and also [the networking engine](https://github.com/holochain/n3h).  You can install and work on core using the built-in mock network following the instructions below, but if you want to actually test out your apps using the real networking, you will have to install [the networking component](https://github.com/holochain/n3h) following the instructions in the readme there.  (Note: please see the instructions in the [`hc` command-line tool readme](./cmd/README.md#using-real-networking) or the [container readme](./container/README.md#using-real-networking) for how to configure the tools to use the find and activate the networking component.
```
Comment (Connoropolous) [inline README.md]:
```suggestion
There are two components needed currently to run Holochain applications, the core (what's in this repo) and also [the networking engine](https://github.com/holochain/n3h).  You can install and work on core using the built-in mock network following the instructions below, but if you want to actually test out your apps using the real networking, you will have to install [the networking component](https://github.com/holochain/n3h) following the instructions in the readme there.  (Note: please see the instructions in the [`hc` command-line tool readme](./cmd/README.md#using-real-networking) or the [container readme](./container/README.md#using-real-networking) for how to configure the tools to use and activate the networking component.
```

Comment (Connoropolous) [reply README.md]: issue with grammar near the end

**Pull request merged**